v3/tutorials/02-proof-of-existence/index.mdx

```diff
@@ -378,14 +341,38 @@ To implement this logic in the proof-of-existence pallet:
 
 1. Save your changes and close the file.
 
-1. Check that your code compiles by running the following command:
+At this point you have a completed pallet! Now to _use_ the pallet, we must correctly configure it in your runtime. 
+
+## Include `MaxBytesInHash` runtime configuration
+
+You should be curious that the proof-of-existence pallet uses the `BoundedVec<u8, T::MaxBytesInHash>` type for proofs, but thus far we have no concrete notion of what `MaxBytesInHash` is.
+This constant should be set in the runtime to something reasonable for use in your blockchain.
+One very typical hash type used in many web3 applications is a [CID](https://github.com/multiformats/cid), and the V1 instance of these is uniformly `60 bytes` in length.
+So here we specify hashes will be of this length or less in the runtime:
+
+1. Open the `runtime/src/lib.rs` file in a text editor.
+
+1. Update the `pallet_template::Config` block to include:
+
+   ```rust
+   /// Configure the pallet-template in pallets/template.
+   impl pallet_template::Config for Runtime {
+   	type Event = Event;
+   	type MaxBytesInHash = frame_support::traits::ConstU32<60>;
+   }
+   ```
+
+1. Save your changes and close the file.
+
+1. (Optionally) check that your code compiles by running the following command:
 
    ```bash
    cargo check -p node-template-runtime
    ```
 
-There is a full Node Template solution [here](https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions/proof-of-existence) to use as a reference if you're stuck.
-Check the [commit diff from the base template](https://github.com/substrate-developer-hub/substrate-node-template/commit/773d72752f3598e5d405b48c6f716f4153c95070#diff-f191d0eb0afa874a64fb9be550a275f957e220b9076c87d4085c2797ca4e310c) for the exact changes.
+<!-- FIXME:  -->
+<!-- There is a full Node Template solution [here](https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions/proof-of-existence) to use as a reference if you're stuck.
+Check the [commit diff from the base template](https://github.com/substrate-developer-hub/substrate-node-template/commit/773d72752f3598e5d405b48c6f716f4153c95070#diff-f191d0eb0afa874a64fb9be550a275f957e220b9076c87d4085c2797ca4e310c) for the exact changes. -->
```

Comment:
This could be updated to the branches tested on, or omitted from now on as it's something to maintain. Should be addressed in #478 long term